### ISSUE TITLE
Client looses Moderator or Trusted status on Logout.

### ISSUE THREAD

**Issue by DigitalMike** [zelloptt/zello-channel-api#42]
I'm unable to keep the client trusted within a channel, it also removes any moderator or admin privileges when logging out.

**Comment by megamk** [comment]
hello @DigitalMike 
I wasn't able to reproduce the issue reported.
I connected as a `user` to a `channel` using Sdk. Granted admin permissions to a `user`, then reconnected again using the sdk and had admin permissions. 
Could you please provide steps to reproduce
Thank you

**Comment by Allanksr** [comment]
Yes, but it really happens, just once it worked to make the user trustworthy, but 99% of the time, you make the user trustworthy and the same goes back to the previous state.
Using Browser Javascript API reference and Android SDK API reference.

**Comment by richdunajewski** [comment]
I saw this behavior as well on a channel that anyone could join but only trusted members could talk. I had to keep granting access to the client and it would lose trusted status next time I ran the code and joined the channel again. 

Seems like there's some issue in general with access control not working quite right.
